src/main/java/core/basesyntax/interfaces/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface Drawable {
+    String draw(Object o);

```

Comment:
Object should draw itself, so no need of this arg
```suggestion
    String draw();
```

Reply:
is **return this.toString** the right solution?